src/components/BookDetails/BookDetails.jsx

```diff
@@ -15,10 +15,25 @@ export default function BookDetails({
 			<div className="book-details">
 				<p className="book-title">{title}</p>
 				<p className="book-author">
-					By {authors} ({publishDate})
+					{Array.isArray(authors)
```

Comment:
Can we fix the model here to always make "authors" and "categories" an array of strings? We shouldn't have data that can be multiple different types. If there is only one author or category, it should be an array of length 1.

Might be worth double checking with Ievgeniia and Ivan to make sure this is the expected model.